server/auth.js

```diff
@@ -3,91 +3,117 @@
  */
 
 const fetch = require("node-fetch");
+module.exports = {
```

Comment:
So the initial reason I made these functions in a separate file was because I wanted to try to test them separately, and reuse them in other places as needed (i.e.: the lambda). I think the best way to do that is to move this out of `./server` and into something like `./lib`, and then I can zip it into the lambda code as well, since it's decoupled from the server stuff.

Reply:
Also, that way we can make it into more an actual common lib, rather than just some code I threw together and shoved into a module

Reply:
will refac into ./lib